.travis.yml

```diff
@@ -14,6 +14,7 @@ jdk:
 - openjdk8
 services:
   - postgresql
+  - mysql
```

Comment:
mysql is installed in "before_install", is the mysql service here still necessary?

Reply:
I have removed `mysql` from service.